cla_public/apps/checker/fields.py

```diff
@@ -161,10 +161,22 @@ def __init__(self, label=None, validators=None, min_val=0, max_val=9999999999, *
         self.min_val = min_val
         self.max_val = max_val
 
+    def convert_amount_to_utf8(self, amount):
+        # Covert pounds side of the input into UTF-8 format
+        try:
+            amount = amount.decode("utf-8")
+        except UnicodeEncodeError:
+            # Input is already in UTF-8 format
+            pass
+
+        return amount
+
     def process_formdata(self, valuelist):
         if valuelist:
             pounds, _, pence = valuelist[0].strip().partition(".")
-            pounds = re.sub(r"[\s,]+", "", pounds)
+            pounds = self.convert_amount_to_utf8(pounds)
+            # xa3 is the numeric character reference for the pound sign
+            pounds = re.sub(r"^(£|\xa3)|[\s,]+", "", pounds)
```

Comment:
I'm not sure why we need the "£" when we do the try/catch decoding to get everything in the same encoding. When does a string reach here with a literal £ in?